agent/config/config_test.go

```diff
@@ -93,8 +93,12 @@ func TestEnvironmentConfig(t *testing.T) {
 	os.Setenv("ECS_IMAGE_CLEANUP_INTERVAL", "2h")
 	os.Setenv("ECS_IMAGE_MINIMUM_CLEANUP_AGE", "30m")
 	os.Setenv("ECS_NUM_IMAGES_DELETE_PER_CYCLE", "2")
+	os.Setenv("ECS_INSTANCE_ATTRIBUTES", "{\"my_attribute\": \"testing\"}")
 
-	conf := environmentConfig()
+	conf, err := environmentConfig()
+	if err != nil {
```

Comment:
These tests haven't been converted yet, but since you're here please do so.

Add to the imports `github.com/stretchr/testify/assert`.  Then use it like this:

```go
conf, err := environmentConfig()
assert.Nil(t, err, "environmentConfig() returned an unexpected error")
assert.Equal(t, "myCluster", conf.Cluster, "Wrong value for cluster")
```